### PR TITLE
Revert "Set next development iteration as 1.1.0-m1-SNAPSHOT"

### PR DESCRIPTION
**Pull request by renuka-fernando**
Reverts wso2/product-microgateway#2586

### CODE REVIEW

**Comment by renuka-fernando** [comment]
Will redo this https://github.com/wso2/product-microgateway/pull/2586/commits/ded1c4aaff63bb7bd5eafa74e5bf2cd320106af8

File: integration/test-integration/pom.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
         <dependency>
             <groupId>org.wso2.am.choreo.connect</groupId>
             <artifactId>mock-backend-server</artifactId>
-            <version>${project.parent.version}</version>
+            <version>1.0.0-rc3-SNAPSHOT</version>
```
Comment (praminda) [inline integration/test-integration/pom.xml:228]:
```suggestion
            <version>${project.parent.version}</version>
```

File: integration/test-integration/pom.xml [hunk 2]
```diff
@@ -244,7 +244,7 @@
         <dependency>
             <groupId>org.wso2.am.choreo.connect</groupId>
             <artifactId>org.wso2.choreo.connect.enforcer</artifactId>
-            <version>${project.parent.version}</version>
+            <version>1.0.0-rc3-SNAPSHOT</version>
```
Comment (praminda) [inline integration/test-integration/pom.xml:247]:
```suggestion
            <version>${project.parent.version}</version>
```

File: integration/test-integration/src/test/resources/dockerCompose/backend-service-tls.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
       - "5006:5006"
   mockBackend:
-    image: wso2/choreo-connect-mock-backend:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-mock-backend:1.0.0-rc2-SNAPSHOT
```
Comment (praminda) [inline integration/test-integration/src/test/resources/dockerCompose/backend-service-tls.yaml:3]:
```suggestion
    image: wso2/choreo-connect-mock-backend:1.0.0-rc3-SNAPSHOT
```

File: integration/test-integration/src/test/resources/dockerCompose/backend-service-with-tls-and-network.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
       - "5006:5006"
   mockBackend:
-    image: wso2/choreo-connect-mock-backend:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-mock-backend:1.0.0-rc2-SNAPSHOT
```
Comment (praminda) [inline integration/test-integration/src/test/resources/dockerCompose/backend-service-with-tls-and-network.yaml:3]:
```suggestion
    image: wso2/choreo-connect-mock-backend:1.0.0-rc3-SNAPSHOT
```

File: integration/test-integration/src/test/resources/dockerCompose/backend-service.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
       - "5006:5006"
   mockBackend:
-    image: wso2/choreo-connect-mock-backend:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-mock-backend:1.0.0-rc2-SNAPSHOT
```
Comment (praminda) [inline integration/test-integration/src/test/resources/dockerCompose/backend-service.yaml:3]:
```suggestion
    image: wso2/choreo-connect-mock-backend:1.0.0-rc3-SNAPSHOT
```

File: integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml [hunk 6]
```diff
@@ -7,7 +7,7 @@ networks:
       name: apim_and_cc
 services:
   router:
-    image: wso2/choreo-connect-router:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-router:1.0.0-rc2-SNAPSHOT
```
Comment (praminda) [inline integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml:10]:
```suggestion
    image: wso2/choreo-connect-router:1.0.0-rc3-SNAPSHOT
```

File: integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml [hunk 7]
```diff
@@ -35,7 +35,7 @@ services:
     networks:
       - cc_internal
   adapter:
-    image: wso2/choreo-connect-adapter:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-adapter:1.0.0-rc2-SNAPSHOT
```
Comment (praminda) [inline integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml:38]:
```suggestion
    image: wso2/choreo-connect-adapter:1.0.0-rc3-SNAPSHOT
```

File: integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml [hunk 8]
```diff
@@ -58,7 +58,7 @@ services:
       - cc_internal
       - apim_and_cc
   enforcer:
-    image: wso2/choreo-connect-enforcer:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-enforcer:1.0.0-rc2-SNAPSHOT
```
Comment (praminda) [inline integration/test-integration/src/test/resources/dockerCompose/cc-in-common-network-docker-compose.yaml:61]:
```suggestion
    image: wso2/choreo-connect-enforcer:1.0.0-rc3-SNAPSHOT
```

File: resources/docker-compose/apim/docker-compose.yaml [hunk 9]
```diff
@@ -14,7 +14,7 @@ services:
     volumes:
       - ./conf/deployment.toml:/home/wso2carbon/wso2am-4.0.0/repository/conf/deployment.toml
   router:
-    image: wso2/choreo-connect-router:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-router:1.0.0
```
Comment (renuka-fernando) [inline resources/docker-compose/apim/docker-compose.yaml:17]:
```suggestion
    image: wso2/choreo-connect-router:1.0.0-rc3-SNAPSHOT
```

File: resources/docker-compose/apim/docker-compose.yaml [hunk 10]
```diff
@@ -43,7 +43,7 @@ services:
       - adapter
       - enforcer
   adapter:
-    image: wso2/choreo-connect-adapter:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-adapter:1.0.0
```
Comment (renuka-fernando) [inline resources/docker-compose/apim/docker-compose.yaml:46]:
```suggestion
    image: wso2/choreo-connect-adapter:1.0.0-rc3-SNAPSHOT
```

File: resources/docker-compose/apim/docker-compose.yaml [hunk 11]
```diff
@@ -63,7 +63,7 @@ services:
     links:
       - apim
   enforcer:
-    image: wso2/choreo-connect-enforcer:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-enforcer:1.0.0
```
Comment (renuka-fernando) [inline resources/docker-compose/apim/docker-compose.yaml:66]:
```suggestion
    image: wso2/choreo-connect-enforcer:1.0.0-rc3-SNAPSHOT
```

File: resources/docker-compose/docker-compose.yaml [hunk 12]
```diff
@@ -1,7 +1,7 @@
 version: "3.7"
 services:
   router:
-    image: wso2/choreo-connect-router:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-router:1.0.0
```
Comment (renuka-fernando) [inline resources/docker-compose/docker-compose.yaml:4]:
```suggestion
    image: wso2/choreo-connect-router:1.0.0-rc3-SNAPSHOT
```

File: resources/docker-compose/docker-compose.yaml [hunk 13]
```diff
@@ -31,7 +31,7 @@ services:
       - adapter
       - enforcer
   adapter:
-    image: wso2/choreo-connect-adapter:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-adapter:1.0.0
```
Comment (renuka-fernando) [inline resources/docker-compose/docker-compose.yaml:34]:
```suggestion
    image: wso2/choreo-connect-adapter:1.0.0-rc3-SNAPSHOT
```

File: resources/docker-compose/docker-compose.yaml [hunk 14]
```diff
@@ -50,7 +50,7 @@ services:
       - "18000:18000"
       - "9843:9843"
   enforcer:
-    image: wso2/choreo-connect-enforcer:1.1.0-m1-SNAPSHOT
+    image: wso2/choreo-connect-enforcer:1.0.0
```
Comment (renuka-fernando) [inline resources/docker-compose/docker-compose.yaml:53]:
```suggestion
    image: wso2/choreo-connect-enforcer:1.0.0-rc3-SNAPSHOT
```

File: resources/k8s-artifacts/choreo-connect/adapter-deployment.yaml [hunk 15]
```diff
@@ -47,7 +47,7 @@ spec:
             - mountPath: /home/wso2/conf/log_config.toml
               subPath: log_config.toml
               name: logconfig-toml-vol
-          image: wso2/choreo-connect-adapter:1.1.0-m1-SNAPSHOT
+          image: wso2/choreo-connect-adapter:1.0.0
```
Comment (renuka-fernando) [inline resources/k8s-artifacts/choreo-connect/adapter-deployment.yaml:50]:
```suggestion
          image: wso2/choreo-connect-adapter:1.0.0-rc3-SNAPSHOT
```

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml [hunk 16]
```diff
@@ -42,7 +42,7 @@ spec:
               name: log4j2-vol
             - mountPath: /home/wso2/lib/dropins
               name: dropins-vol
-          image: wso2/choreo-connect-enforcer:1.1.0-m1-SNAPSHOT
+          image: wso2/choreo-connect-enforcer:1.0.0
```
Comment (renuka-fernando) [inline resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml:45]:
```suggestion
          image: wso2/choreo-connect-enforcer:1.0.0-rc3-SNAPSHOT
```

File: resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml [hunk 17]
```diff
@@ -108,7 +108,7 @@ spec:
               name: router-keystore-vol
             - mountPath: /home/wso2/security/truststore
               name: router-truststore-vol
-          image: wso2/choreo-connect-router:1.1.0-m1-SNAPSHOT
+          image: wso2/choreo-connect-router:1.0.0
```
Comment (renuka-fernando) [inline resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml:111]:
```suggestion
          image: wso2/choreo-connect-router:1.0.0-rc3-SNAPSHOT
```

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211216.13

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211216.15

**Pull request merged**